lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -725,6 +725,8 @@ sub _applicable_upgrade_tests {
                   LedgerSMB::Upgrade_Tests->get_tests;
 }
 
+my @checks;
```

Comment:
This line makes me think this PR does not yet include the insights from #3351.

Reply:
Fixed